.travis.yml

```diff
@@ -12,16 +12,14 @@ install:
   - curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/v0.9.0/kind-$(uname)-amd64
   - chmod +x ./kind
   - mv ./kind ${GOPATH}/bin
-script:
-  - git-validation -run DCO,short-subject
-  - go fmt $(go list ./... | grep -v vendor) | grep -v "api.pb.go" | wc -l | grep "^0";
-  - make docker-proto
-  - git diff $(find . -name "*.pb.*go" -o -name "api.swagger.json" | grep -v vendor) | wc -l | grep "^0"
-  - hack/check-api-version.sh
-  - git grep -rw GPL vendor | grep LICENSE | egrep -v "yaml.v2" | wc -l | grep "^0"
-  - bash hack/docker-integration-test.sh
-  - bash hack/csi-sanity-test.sh
-  - make install verify
```

Comment:
is `make install verify` run in the new parallel jobs?

Reply:
yeah, we run `make pr-verify now`. I broke `verify` down into separate targets.

`make install` is ran as part of `osd-tests`